doc/GUIDE.md

```diff
@@ -986,6 +988,21 @@ A complete list of all the supported command line options is given below:
 | `--include-[function,constant,struct,union,typedef,var]<String>` | Include a symbol of the given name and kind in the generated bindings. When one of these options is specified, any symbol that is not matched by any specified filters is omitted from the generated bindings. |
 | `--version`                                                  | print version information and exit |
 
+Jextract accepts one or more header files. When multiple header files are specified,
```

Comment:
I suggest adding a header above this section to separate it from the table (like we currently have for the section on clang arguments below)

Reply:
Hmm.. not sure what would be the name. Perhaps move this para above the table?

Reply:
Discussed offline: let's keep it like this